examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,73 @@
+"""
+Extracting resting-state signals from different atlases for comparison
```

Comment:
Comparing connectomes on different reference parcellation atlases

Reply:
Given that we are going to demo both functions in this example, remove parcellation from above.